src/style.css

```diff
@@ -0,0 +1,162 @@
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  font-family: Roboto, sans-serif;
+}
+
+.card__informacion {
+  margin: 0 16px 16px;
+  width: 166px;
+}
+
+.card {
+  width: 198px;
+  height: 406px;
+  top: 1px;
+  left: 1px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.card__photo {
+  margin: 32px 19px 40px;
+  width: 160px;
+  height: 134px;
+  background-position: center;
+  background-size: 100% 100%;
+  background-image: url("./images/imac.jpag");
```

Comment:
There is a typo in the background image path. The file extension should be `.jpg` instead of `.jpag`.